models/handpose_estimation_mediapipe/demo.py

```diff
@@ -214,6 +344,29 @@ def draw_lines(image, landmarks, is_draw_point=True, thickness=2):
                 handpose = handpose_detector.infer(frame, palm)
                 if handpose is not None:
                     hands = np.vstack((hands, handpose))
+
+                    for idx, handpose in enumerate(hands):
+                        conf = handpose[-1]
+                        bbox = handpose[0:4].astype(np.int32)
+                        handedness = handpose[-2]
+
+                        landmarks_screen = handpose[4:67].reshape(21, 3).astype(np.int32)
+                        landmarks_word = handpose[67:130].reshape(21, 3)
+
+                        landmarks = landmarks_word[:, [0, 1]]
+                        landmarks = (landmarks * 1000 + 100).astype(np.int32)
+
+                        hand_local = []
+                        for i in range(21):
+                            x = landmarks[i][0] * frame.shape[1]
+                            y = landmarks[i][1] * frame.shape[0]
+                            hand_local.append((x, y))
+
+                        if hand_local:
+                            gesture = h_gesture(hand_local)
+                        if gesture is not None:
+                            tmp = dict[gesture]
+                            cv.putText(frame, gesture, (1000, 100), 0, 1.3, (0, 0, 255), 3)
```

Comment:
The code about visualization needs to be placed in `visualize` function.

Reply:
One more suggestion here is name the function something like `gesture_classification` to make it more meaningful.